threedigrid_builder/grid/cross_section_definitions.py

```diff
@@ -112,7 +132,7 @@ def tabulate_builtin(shape, width, height):
             f"Unable to parse cross section definition width (got: '{width}')."
         )
 
-    return shape, width, None, None
+    return shape, width, None, None, None
```

Comment:
Can you adjust the docstring?

Reply:
I think this still needs to be done

Reply:
This is the case for all tabulate functions defined here I think:
```python
tabulators = {
    CrossSectionShape.CLOSED_RECTANGLE: tabulate_closed_rectangle,
    CrossSectionShape.RECTANGLE: tabulate_builtin,
    CrossSectionShape.CIRCLE: tabulate_builtin,
    CrossSectionShape.EGG: tabulate_egg,
    CrossSectionShape.TABULATED_RECTANGLE: tabulate_tabulated,
    CrossSectionShape.TABULATED_TRAPEZIUM: tabulate_tabulated,
    CrossSectionShape.TABULATED_YZ: tabulate_yz,
    CrossSectionShape.INVERTED_EGG: tabulate_inverted_egg,
}
```